### DISCUSSION TITLE
Identity validation config example is invalid

### DISCUSSION THREAD

**Discussion by HannesJo0139** [authelia/authelia#7124] [Documentation Feedback]
The config example at [https://www.authelia.com/configuration/identity-validation/introduction/](url) provides an invalid key `identity_validation.reset_password.expiration` that should be updated to `identity_validation.reset_password.jwt_lifespan`

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

To address the issue with the invalid configuration key `identity_validation.reset_password.expiration` that should be `identity_validation.reset_password.jwt_lifespan`, it's important to refer to the official Authelia documentation and resources for the correct configuration parameters. Based on the provided context, the correct key for configuring the lifespan of the JWT for reset password flows in Authelia is indeed `jwt_lifespan`, as detailed in the [Reset Password Identity Validation](https://www.authelia.com/configuration/identity-validation/reset-password) section of the Authelia documentation.

This key is used to specify the duration for which the JWT is valid after being initially generated, ensuring that users cannot perform a reset password flow without first being adequately identified. The correct configuration should look something like this in the `configuration.yml` file:

```yaml
identity_validation:
  reset_password:
    jwt_lifespan: '5 minutes'
    jwt_algorithm: 'HS256'
    jwt_secret: 'your_jwt_secret_here'
```

It's crucial to ensure that the configuration aligns with the latest documentation and examples provided by Authelia to avoid issues with deprecated or incorrect configuration keys. If you encounter a configuration key that is not recognized or deprecated, it's advisable to consult the latest version of the documentation or the Authelia GitHub repository for updates or clarifications.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment]
Thanks, this has been adjusted.
